### DISCUSSION TITLE
How to reference JS file in the plugin?

### DISCUSSION THREAD

**Discussion by ferdware** [givanz/Vvveb#209] [General]
Here is the folder structure of the test-plugin ( according to the plugin documentation )
[
test-plugin
- admin
- public
  - js
Index.html
]

I have a test onclick function in index.html
<div><button type="button" onclick="test_plugin_test()">Test Button</button></div>

I added to the top of index.html
<head data-v-copy-from="blank.html,head">
   <script src="/public/js/plugins.js" ></script>
</head>

I noticed my script entry gets overwritten when I view the raw html.

What’s the best way to make reference to the JS files in the plugin from index.html?

Thank You again.

**Comment by givanz** [comment]
The base for all pages is set in the `<head>` with 

```html
<head>
	<base href="/themes/currently-active-theme/">
</head>
```

Because of this to refrence files directly in `/public/` folder you need to prefix the path with `../../` 

To include a `test.js` file located in `/vvveb/public/plugins/test-plugin/test.js` you should use

```html
<script src="../../plugins/test-plugin/test.js"></script>
```

**Comment by ferdware** [comment]
Thank You!
